lib/course_planner/users.ex

```diff
@@ -44,6 +47,19 @@ defmodule CoursePlanner.Users do
     |> @notifier.notify_later()
   end
 
+  def update_notifications(user) do
+    Multi.new()
+    |> delete_notifications(user.notifications)
```

Comment:
what do you think of leaving lines 52 and 53 in the same level of abstraction? It may be something like
```elixir
Multi.new()
  |> Multi.delete_all(Notification, notifications_to_delete(params))
  |> Multi.update(User, Changeset.change(user, notified: Ecto.Date.utc()))
  |> Repo.transaction()
```
or
```elixir
Multi.new()
  |> delete_notifications(user.notifications)
  |> update_last_notified(params)
  |> Repo.transaction()
```